petl/transform/maps.py

```diff
@@ -70,7 +71,7 @@ def __init__(self, source, mappings=None, failonerror=False,
             self.mappings = OrderedDict()
         else:
             self.mappings = mappings
-        self.failonerror = failonerror
+        self.failonerror = failonerror or config.failonerror
```

Comment:
Same comment as above, need to consider how user can override the config value, regardless of what the config value is.